wgsl/index.bs

```diff
@@ -1309,57 +1309,142 @@ repeatedly replacing a subsequence with its homoglyph.)
+* <a for=attribute lt=const>`'const'`</a>
+* <a for=attribute lt=fragment>`'fragment'`</a>
+* <a for=attribute lt=group>`'group'`</a>
+* <a for=attribute lt=id>`'id'`</a>
+* <a for=attribute lt=interpolate>`'interpolate'`</a>
+* <a for=attribute lt=invariant>`'invariant'`</a>
+* <a for=attribute lt=location>`'location'`</a>
+* <a for=attribute lt=size>`'size'`</a>
+* <a for=attribute lt=vertex>`'vertex'`</a>
+* <a for=attribute lt=workgroup_size>`'workgroup_size'`</a>
+
+### Built-In Value Names ### {#builtin-value-names}
 
 A <dfn noexport>built-in value name-token</dfn> is a [=token=] used in the name of a [=built-in value=].
 The spelling of the token may be the same as an [=identifier=] but does not [=resolves|resolve to=] a declared object.
-The token must not be a [=keyword=] or [=reserved word=].
-
-See [[#builtin-inputs-outputs]].
+The token must not be a [=keyword=] or [=reserved word=]. See [[#builtin-inputs-outputs]].
 
 <pre class=include>
 path: syntax/builtin_value_name.syntax.bs.include
 </pre>
 
-## Diagnostic Rule Names ## {#diagnostic-rule-names}
+The [=built-in value=] names are:
+
+* <a for="built-in values" lt=vertex_index>`'vertex_index'`</a>
+* <a for="built-in values" lt=instance_index>`'instance_index'`</a>
+* <a for="built-in values" lt=position>`'position'`</a>
+* <a for="built-in values" lt=front_facing>`'front_facing'`</a>
+* <a for="built-in values" lt=frag_depth>`'frag_depth'`</a>
+* <a for="built-in values" lt=sample_index>`'sample_index'`</a>
+* <a for="built-in values" lt=sample_mask>`'sample_mask'`</a>
+* <a for="built-in values" lt=local_invocation_id>`'local_invocation_id'`</a>
+* <a for="built-in values" lt=local_invocation_index>`'local_invocation_index'`</a>
+* <a for="built-in values" lt=global_invocation_id>`'global_invocation_id'`</a>
+* <a for="built-in values" lt=workgroup_id>`'workgroup_id'`</a>
+* <a for="built-in values" lt=num_workgroups>`'num_workgroups'`</a>
+
+### Diagnostic Rule Names ### {#diagnostic-rule-names}
 
 A <dfn noexport>diagnostic name-token</dfn> is a [=token=] used in the name of a diagnostic [=diagnostic/triggering rule=].
 The spelling of the token may be the same as an [=identifier=] but does not [=resolves|resolve to=] a declared object.
-The token must not be a [=keyword=] or [=reserved word=].
-
-See [[#diagnostics]].
+The token must not be a [=keyword=] or [=reserved word=]. See [[#diagnostics]].
 
 <pre class=include>
 path: syntax/diagnostic_name_token.syntax.bs.include
 </pre>
 
-## Interpolation Type Names ## {#interpolation-type-names}
+The [=diagnostic/triggering rule|diagnostic rule=] names are:
 
-An <dfn noexport>interpolation type name-token</dfn> is a [=token=] used in the name of an [=interpolation type=].
-The spelling of the token may be the same as an [=identifier=] but does not [=resolves|resolve to=] a declared object.
-The token must not be a [=keyword=] or [=reserved word=].
+* <a for=trigger lt=derivative_uniformity>`'derivative_uniformity'`</a>
```

Comment:
Nit: this doesn't feel completely true. We intentionally allow unknown diagnostic rules. Maybe we should say pre-defined here. I'd be ok treating this as pre-existing issue.

Reply:
Better to be a bit more specific already, so I prefixed saying pre-defined, but I do agree a subsequent issue can take a closer look. Thanks.